repository-hbase/src/main/java/hbase/load/BulkLoader.java

```diff
@@ -168,7 +172,8 @@ private int load(String tableNameStr, String referencePeriod, String inputFile,
 
     public static void main(String[] args) {
         try {
-            HBaseConnector connector = new HBaseInMemoryConnector(args[1]);
```

Comment:
Let's remove since we don't require in-memory connection via the executable JAR 😄